app/design/frontend/base/default/template/boltpay/catalog/product/configure_checkout.phtml

```diff
@@ -17,94 +17,52 @@
 
 /* @var $this Bolt_Boltpay_Block_Catalog_Product_Boltpay */
 ?>
-<?php if ($this->isEnabledProductPageCheckout() && $this->isSupportedProductType()): ?>
-<script type="text/javascript">
-var productPageCheckoutSelector = '<?php $this->escapeHtml($this->getProductPageCheckoutSelector()) ?>';
-var order_completed = false;
-var do_checks = 1;
-<?php $productTierPrice = $this->getProductTierPrice(); ?>
-var boltConfigPDP = {
-    _jsonProductCart: <?php echo $this->getCartDataJsForProductPage(); ?>,
-    _tierPrices : <?php echo json_encode($productTierPrice) ?>,
-    getPrice : function (qty) {
-        qty = Number(qty);
-        var i;
-        for (i = this._tierPrices.length; i > 0; --i) {
-            if (qty >= this._tierPrices[i-1]['price_qty']){
-                return this._tierPrices[i-1]['price'];
+<?php if ($this->isEnabledProductPageCheckout()): ?>
+    <?php if (!$this->isSupportedProductType()):?>
+    <script type="text/javascript">
+        var stateOld = {};
+        var stateNew = {};
+        
+        var startBolt = function() {
+            Object.assign(stateOld, spConfig.state);
+            if (JSON.stringify(stateOld) == JSON.stringify(stateNew)) {
+                return;
             }
+            Object.assign(stateNew, spConfig.state);
+            jQuery('#ppcBuyNow').show();
+            jQuery('div.bolt-product-checkout-button').html('');
```

Comment:
Can you change `html` to `empty`. I think it is more faster.
https://stackoverflow.com/questions/17543711/jquery-difference-html-vs-empty